tests/extract/test_sources.py

```diff
@@ -788,52 +788,6 @@ def test_add_transformer_right_pipe() -> None:
         iter([1, 2, 3]) | dlt.resource(lambda i: i * 3, name="lambda")
 
 
-def test_limit_infinite_counter() -> None:
```

Comment:
these three tests were moved to the new location where a couple more tests will be added specifically for the limits